static/panes/compiler.js

```diff
@@ -1950,6 +1950,13 @@ Compiler.prototype.checkForUnwiseArguments = function (optionsArray) {
     var are = unwiseOptions.length === 1 ? ' is ' : ' are ';
     var msg = options + names + are + 'not recommended, as behaviour might change based on server hardware.';
 
+    if (_.contains(optionsArray, '-flto')) {
+        this.alertSystem.notify('Option -flto is being used without Compile to Binary.', {
+            group: 'unwiseOption',
+            collapseSimilar: true,
+        });
+    }
```

Comment:
would be better to have an else-if and the msg inside the other if

Reply:
Correct me if I'm wrong, but changing it to an `else if` would exclude the scenario where both of the branches should evalute to true (maybe this is intentional), as in the case of an `unwiseOptions` (e.g. -march=native) and also -flto being used.

As regards the variable scoping I changed that, and I also noticed that we should raise this warning only if Compile to Binary is actually turned off, therefore I've added this extra condition.

Reply:
I'm not quite sure what our alert system does with two notifies at once; but in the interests of getting this merged I think this is just fine! Thanks for your patience!

Reply:
Lost track of this comment, but yeah that was my exact concern. I guess we will find out :)

Reply:
Sorry for bothering again, and I might sound a little repetitive, but I wanted to underline the fact that at the moment the message is shown also with Compile to Binary turned on. As i mentioned earlier i had already fixed this and i was waiting for @partouf's approval to push the changes. Now since this has been already merged if it is fine for you, it's obviously fine for me, I just wanted you to be aware of it.

Reply:
I see live that theres only 1 message at the time active, -march has preference because it's the last to be notified about, if you remove march, the hint about -flto appears. I guess it's the collapseSimilar that arranges that only the last message will show. Not the clearest from the code, but it works.